test/utils/fixture/fixture.go

```diff
@@ -70,6 +71,7 @@ const (
 	OptionNone          = 0
 	OptionRandomizeName = 1 << iota
 	OptionAvoidCooldown = 2 << iota
+	OptionIsSuite       = 4 << iota
```

Comment:
I was reluctant if these options are meant only for single tests but found it handy instead of passing another parameter to the setupWithOption to indicate a suite setup, @ffromani WDYT about keeping it?

Reply:
we need to review how use `iota` here (mostly my fault)